earthfile2llb/converter.go

```diff
@@ -596,13 +595,34 @@ func (c *Converter) RunExitCode(ctx context.Context, opts ConvertRunOpts) (int,
 
 // RunExpression runs an expression and returns its output. The run is transient - any state created
 // is not used in subsequent commands.
+// e.g. the expression "hello $(cat /data)" gets transformed into /bin/sh -c 'echo "hello $(cat /data)"', before being executed
```

Comment:
Is this comment still accurate?

Reply:
technically yes, but that that occurs in shell.go, so i'll remove this comment.